src/compiler/scale.ts

```diff
@@ -1,26 +1,26 @@
-'use strict';
-require('../globals');
-var util = require('../util'),
-  time = require('./time'),
-  colorbrewer = require('colorbrewer'),
-  interpolate = require('d3-color').interpolateHsl;
+/// <reference path="../colorbrewer.d.ts"/>
```

Comment:
Should put in README when do we use import, when do we use require and when <reference>